src/agentscope/models/openai_model.py

```diff
@@ -401,31 +410,26 @@ def _format_msg_with_url(
 
             return returned_msg
 
-    def format(
-        self,
+    @staticmethod
+    def static_format(
         *args: Union[Msg, Sequence[Msg]],
+        model_name: str,
     ) -> List[dict]:
-        """Format the input string and dictionary into the format that
-        OpenAI Chat API required.
-
-        Args:
```

Comment:
Why remove the `Args` and `Returns` here?

Reply:
Add Args and Returns here.